packages/contractkit/src/identity/claims/attestation-service-url.ts

```diff
@@ -57,10 +57,9 @@ export async function validateAttestationServiceUrl(
 
     const attestationKeyAddress = await accounts.getAttestationSigner(address)
 
-    // Uncomment this once we opt in by specifying an attestation key
```

Comment:
@nambrot can you check if this should be uncommented now?

Reply:
Yup should be good now